src/autopas/utils/SimilarityFunctions.h

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file SimilarityFunctions.h
+ * @author J. Kroll
+ * @date 01.06.2020
+ */
+
+#pragma once
+
+#include "ThreeDimensionalMapping.h"
+#include "autopas/containers/ParticleContainerInterface.h"
+#include "autopas/molecularDynamics/MoleculeLJ.h"
+
+namespace autopas::utils {
+
+/**
+ * calculates homogeneity and max density of given AutoPas simulation
+ *
+ * @tparam Particle
+ * @param autopas
+ * @return  [homogeneity, max_density]
+ */
+template<class Particle>
+double static *calculateHomogeneityAndMaxDensity(const std::shared_ptr<autopas::ParticleContainerInterface<Particle>>& container) {
+  auto *data = new double[2];
```

Comment:
```suggestion
```
Please also use proper variable names, s.t., one can understand what happens. That way some comments are superfluous.

Reply:
solved with commit "use pair instead of pointer"